examples/medplum-patient-intake-demo/src/pages/QuestionnaireCustomizationPage.tsx

```diff
@@ -0,0 +1,59 @@
+import { normalizeErrorString } from '@medplum/core';
+import { Questionnaire } from '@medplum/fhirtypes';
+import { Document, QuestionnaireBuilder, useMedplum, useMedplumProfile, useResource } from '@medplum/react';
+import { useCallback } from 'react';
+import { showNotification } from '@mantine/notifications';
+import { Loading } from '../components/Loading';
+import { useNavigate } from 'react-router-dom';
+import { IconCircleCheck, IconCircleOff } from '@tabler/icons-react';
+
+export function QuestionnaireCustomizationPage(): JSX.Element {
+  const navigate = useNavigate();
+  const medplum = useMedplum();
+  const profile = useMedplumProfile();
+
+  const questionnaireId = 'd613c8ed-222e-4f74-a220-e170a37d34d8';
```

Comment:
This is fine for this PR, but we will have to update the logic here to avoid hard-coded ids in the example application

Reply:
Yes, this was a temporary thing. Will make the change.